layouts/shortcodes/friday_hack_header.html

```diff
@@ -8,4 +8,10 @@
       </strong>
       <br />
     {{ end }}
+    {{ with .Get "rsvp_link" }}
+    <strong>
+      <a href="{{ . }}" target="_blank" rel="noopener noreferrer">Reserve your spot now — Space is limited!</a>
```

Comment:
hmm do you want to phrase it slightly different? like "Reserve your physical spot now — Space is limited!"

Reply:
Yeah sure!

Reply:
Yeah sure!